README.md

```diff
@@ -1 +1,29 @@
+# Kord
+[![Discord](https://img.shields.io/discord/556525343595298817.svg?color=&label=Kord&logo=discord&style=for-the-badge)](https://discord.gg/6jcx5ev)
```

Comment:
You appear to have enabled the Discord server widget. I suggest disabling this and using a generic badge, as it allows bad actors (such as data brokers) to figure out some of the server's members.

![image](https://github.com/user-attachments/assets/621c4021-8dba-41fd-ad54-385b02d6e936)

Reply:
I just copied this from the Kord repo itself

Reply:
can't find the setting on mobile (on vacation rn)